src/libAtomVM/opcodesswitch.h

```diff
@@ -1657,7 +1657,7 @@ term make_fun(Context *ctx, const Module *mod, int fun_index)
                 break;
             }
 
-#ifndef AVM_NO_FP
```

Comment:
Just for the record, I had to move the `#ifdef` inside the `IMPL_EXECUTE_LOOP` because having an `is_float/1` guard in an application will cause the VM to abort with an unimplemented instruction, and I thought it would be better to just jump to the fail label if we are running code with the instruction (e.g., in `io_lib`).

Reply:
good idea, I agree with that.